packages/@expo/cli/src/prebuild/__tests__/fixtures/contrived-template/macos/Podfile

```diff
@@ -0,0 +1,2 @@
+# This file is a mock just for testing template renaming.
```

Comment:
recommend generating these files in a temp dir instead of adding all these files to git.

Reply:
I've tried implementing that in [this](https://github.com/expo/expo/commit/d0cdc7a4e046383c594aba7ab7c5e3cac75ee9a4) branch, but while it works fine in `create-expo`, I can't get the same code working in `@expo/cli` – I think some Jest mocking must be interfering with my setup code (which relies on `fs/promises`) or something, but I can't figure it out.

If (as per your [other comment](https://github.com/expo/expo/pull/30309#discussion_r1674576684)) you're happy to accept it without tests, though, I'll just throw in the towel and rip out the changes to the tests and fixtures. I've pushed that in the commit `chore(cli,create-expo): revert changes to fixtures and tests` now.